source/grammar.tex

```diff
@@ -10,8 +10,8 @@
 In particular, the grammar described here accepts
 a superset of valid \Cpp{} constructs.
 Disambiguation rules~(\ref{stmt.ambig}, \ref{dcl.spec}, \ref{class.member.lookup})
-must be applied to distinguish expressions from declarations.
-Further, access control, ambiguity, and type rules must be used
+shall be applied to distinguish expressions from declarations.
+Further, access control, ambiguity, and type rules shall be used
```

Comment:
```suggestion
are applied to distinguish expressions from declarations.
Further, access control, ambiguity, and type rules are used
```

We can't "shall" from an informative Annex, and I think this is supposed to be descriptive rather than introducing a requirement.

Reply:
done